HM2_task3.py

```diff
@@ -0,0 +1,22 @@
+# 3
+
+casino_blacklist = ["Jack Pie", "Erik Man", "Lili Evans"]
+poker_blacklist = ["Patrick Low", "Ann Downstown", "Mari Darret"]
+alcohol_blacklist = ["Albert Second", "Kate Signi", "Iri Mane"]
+
+for x in [casino_blacklist, alcohol_blacklist]:
+    poker_blacklist.extend(x)
+
+print(poker_blacklist)
```

Comment:
actually, all these lists are sets because a person can not be in this list twice. 

And the solution is on of the method of sets